### PR TITLE
Bug 1238530 - Clicker UI top bar. r=dmose

### PR DESCRIPTION
**Pull request by mancas**

### CODE REVIEW

File: shared/js/views.jsx [hunk 1]
```diff
@@ -47,12 +47,12 @@ loop.shared.views = (function(_, mozL10n) {
    * - {String}   scope   Media scope, can be "local" or "remote".
    * - {String}   type    Media type, can be "audio" or "video".
    * - {Function} action  Function to be executed on click.
-   * - {Enabled}  enabled Stream activation status (default: true).
+   * - {Enabled}  muted Stream activation status (default: false).
```
Comment (fcampo) [inline shared/js/views.jsx]: [nit] `{bool}` not `{Enabled}`, plus extra space

File: shared/test/views_test.js [hunk 2]
```diff
@@ -172,57 +274,57 @@ describe("loop.shared.views", function() {
...
       });
 
       TestUtils.Simulate.click(
         comp.getDOMNode().querySelector(".btn-mute-audio"));
 
-      sinon.assert.calledOnce(publishStream);
-      sinon.assert.calledWithExactly(publishStream, "audio", false);
+      sinon.assert.calledOnce(dispatcher.dispatch);
+      sinon.assert.calledWithExactly(dispatcher.dispatch,
+        new sharedActions.SetMute({ type: "audio", enabled: false }));
```
Comment (fcampo) [inline shared/test/views_test.js]: if there's no asserts anymore, we don't need the whole test

File: shared/test/views_test.js [hunk 3]
```diff
@@ -172,57 +274,57 @@ describe("loop.shared.views", function() {
...
       });
 
       TestUtils.Simulate.click(
         comp.getDOMNode().querySelector(".btn-mute-audio"));
 
-      sinon.assert.calledOnce(publishStream);
-      sinon.assert.calledWithExactly(publishStream, "audio", true);
+      sinon.assert.calledOnce(dispatcher.dispatch);
+      sinon.assert.calledWithExactly(dispatcher.dispatch,
+        new sharedActions.SetMute({ type: "audio", enabled: true }));
```
Comment (fcampo) [inline shared/test/views_test.js]: same

File: shared/test/views_test.js [hunk 4]
```diff
@@ -172,57 +274,57 @@ describe("loop.shared.views", function() {
...
       });
 
       TestUtils.Simulate.click(
         comp.getDOMNode().querySelector(".btn-mute-video"));
 
-      sinon.assert.calledOnce(publishStream);
-      sinon.assert.calledWithExactly(publishStream, "video", false);
+      sinon.assert.calledOnce(dispatcher.dispatch);
+      sinon.assert.calledWithExactly(dispatcher.dispatch,
+        new sharedActions.SetMute({ type: "video", enabled: false }));
```
Comment (fcampo) [inline shared/test/views_test.js]: same

File: shared/test/views_test.js [hunk 5]
```diff
@@ -172,57 +274,57 @@ describe("loop.shared.views", function() {
...
       });
 
       TestUtils.Simulate.click(
         comp.getDOMNode().querySelector(".btn-mute-video"));
 
-      sinon.assert.calledOnce(publishStream);
-      sinon.assert.calledWithExactly(publishStream, "video", true);
+      sinon.assert.calledOnce(dispatcher.dispatch);
+      sinon.assert.calledWithExactly(dispatcher.dispatch,
+        new sharedActions.SetMute({ type: "video", enabled: true }));
```
Comment (fcampo) [inline shared/test/views_test.js]: same

File: standalone/content/css/webapp.css [hunk 6]
```diff
@@ -44,78 +44,181 @@ body,
   width: 100px;
 }
 
-/* Standalone Overlay wrapper */
-.standalone-overlay-wrapper {
+/* Standalone Info bar */
+.standalone-info-bar {
+  align-items: center;
+  background: #fbfbfb;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, .25);
```
Comment (mikedeboer) [inline standalone/content/css/webapp.css]: nit: rgba(0,0,0,.25)

File: standalone/content/css/webapp.css [hunk 7]
```diff
@@ -44,78 +44,181 @@ body,
...
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.standalone-info-bar > .hello-logo:after {
+  content: "";
+  border-left: solid 1px #c1c1c1;
+  bottom: 0;
+  display: block;
+  /* keep in sync with .hello-logo width and margins */
```
Comment (mikedeboer) [inline standalone/content/css/webapp.css]: nit: start comments with a capital and end with a dot (.).

File: standalone/content/css/webapp.css [hunk 8]
```diff
@@ -44,78 +44,181 @@ body,
...
-  left: 0;
   top: 0;
 }
 
-html[dir="rtl"] .standalone-overlay-wrapper > .hello-logo {
-  right: 0;
-  left: auto;
+html[dir="rtl"] .standalone-info-bar > .hello-logo:after {
+  left: initial;
+  /* keep in sync with .hello-logo width and margins */
```
Comment (mikedeboer) [inline standalone/content/css/webapp.css]: nit: same.

File: standalone/content/js/standaloneRoomViews.jsx [hunk 9]
```diff
@@ -394,6 +393,138 @@ loop.standaloneRoomViews = (function(mozL10n) {
...
+    },
+
+    render: function() {
+      switch (this.props.room.roomState) {
+        case ROOM_STATES.ENDED:
+        case ROOM_STATES.READY:
+        case ROOM_STATES.FULL:
+        case ROOM_STATES.FAILED:
+        case ROOM_STATES.INIT:
+        case ROOM_STATES.GATHER: {
```
Comment (mikedeboer) [inline standalone/content/js/standaloneRoomViews.jsx]: nit: We don't enclose case body blocks with curlies, please remove them.

File: standalone/content/js/standaloneRoomViews.jsx [hunk 10]
```diff
@@ -394,6 +393,138 @@ loop.standaloneRoomViews = (function(mozL10n) {
...
+          return (
+            <div className="context-info">
+              <ToSView dispatcher={this.props.dispatcher} />
+            </div>
+          );
+        }
+        case ROOM_STATES.MEDIA_WAIT:
+        case ROOM_STATES.JOINING:
+        case ROOM_STATES.JOINED:
+        case ROOM_STATES.SESSION_CONNECTED: {
```
Comment (mikedeboer) [inline standalone/content/js/standaloneRoomViews.jsx]: nit: same.

File: standalone/content/css/webapp.css [hunk 11]
```diff
@@ -44,78 +44,181 @@ body,
...
+  content: url("/shared/img/video-muted-darkgrey.svg");
+}
+
+.standalone-info-bar > .media-control-buttons > .general-support-url {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+/* highlight on hover */
```
Comment (mikedeboer) [inline standalone/content/css/webapp.css]: nit: please format comment correctly.

File: standalone/content/css/webapp.css [hunk 12]
```diff
@@ -44,78 +44,181 @@ body,
...
+  background-color: rgba(51, 51, 51, 0.1);
+}
+
+.general-support-url > .icon-help {
+  background: transparent url("../shared/img/glyph-help-no-circle-16x16.svg") no-repeat;
+  height: 16px;
+  width: 16px;
+ }
+
+/* account for the height of the standalone infobar */
```
Comment (mikedeboer) [inline standalone/content/css/webapp.css]: nit: same.

**Comment by mikedeboer** [comment]
So this is r+ from me code-wise, except that I need to see it in action still. I'll do that tomorrow.

**Pull request merged**